frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -178,6 +178,11 @@ const plugin: Plugin<ConsumedExtensions> = [
             throw e;
           });
       },
+      accessReview: {
+        group: models.OCSServiceModel.apiGroup,
+        resource: models.OCSServiceModel.plural,
+        verb: 'patch',
```

Comment:
You need the resource name and namespace. You're checking if you can patch every OCS service in all namespaces.

Reply:
ack